test/integration/db_integration_test.go

```diff
@@ -110,22 +112,28 @@ func (a *dbIntegrationSuite) TestIntegrationWithDb_GetRecords() {
 }
 
 func (a *dbIntegrationSuite) TestIntegrationWithDb_GetJetDrops() {
+	pulse := gen.PulseNumber()
```

Comment:
before these changes records belonged to pulses*2 (expRecordsJet1+ expRecordsJet2), so each record set consisted of `pulses` count.

Now there is one pulse for all records

I think that instead of generating pulse once and then assigning it to the records you should collect pulses from all records to array with `Record.ID.Pulse()` and then get records from DB using that pulses values

Reply:
good point, rewrote it